src/bin/server/transcoder/targets.rs

```diff
@@ -0,0 +1,65 @@
+use std::ffi::OsStr;
+use std::path::Path;
+
+use core::fmt;
+
+#[derive(Debug, Clone)]
+pub enum TranscodeTarget {
+    MP3V0,
+    AACV5,
+}
+
+impl TranscodeTarget {
+    pub fn get_ffmpeg_args<'a>(
+        &self,
+        input_file: &'a Path,
+        output_file: &'a Path,
+    ) -> Vec<&'a OsStr> {
+        match self {
+            TranscodeTarget::MP3V0 => vec![
```

Comment:
Is there a better way to do this?

Reply:
You could do a `format!()` and then split on the spaces, but that might require converting from OsStr to String and back, which could lose data, and the paths might have spaces in them. Since you're using two different types and converting them to OsStr (`&str` and `&Path`), you will have to do some sort of conversion on each one to get it into the final type.

Reply:
So no?